src/seat.c

```diff
@@ -143,6 +143,25 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 		wlr_log(WLR_INFO, "dwt configured");
 		libinput_device_config_dwt_set_enabled(libinput_dev, dc->dwt);
 	}
+
+	if (dc->send_events_mode < 0) {
+	} else if (dc->send_events_mode >= LIBINPUT_CONFIG_SEND_EVENTS_ENABLED) {
+		uint32_t possible_modes =
+			libinput_device_config_send_events_get_modes(libinput_dev);
+		enum libinput_config_status status;
+		if (dc->send_events_mode == LIBINPUT_CONFIG_SEND_EVENTS_ENABLED
+			|| possible_modes & dc->send_events_mode) {
+			status = libinput_device_config_send_events_set_mode(libinput_dev,
+				dc->send_events_mode);
+			if (status != LIBINPUT_CONFIG_STATUS_SUCCESS) {
+				wlr_log(WLR_ERROR, "Failed to set events mode.");
+			} else {
+				wlr_log(WLR_INFO, "Send events mode set successfully!");
+			}
```

Comment:
You can’t have two else :smile:

I think it would be tidier here to do a switch (status)… and handle the three enums (success, unsupported, invalid)

https://wayland.freedesktop.org/libinput/doc/latest/api/group__config.html#ga33ff306ed8505129adce4c092a3ed175